pkg/client/clientset/versioned/clientset.go

```diff
@@ -31,6 +32,9 @@ type Interface interface {
 	CephV1alpha1() cephv1alpha1.CephV1alpha1Interface
 	// Deprecated: please explicitly pick a version if possible.
 	Ceph() cephv1alpha1.CephV1alpha1Interface
+	NfsV1alpha1() nfsv1alpha1.NfsV1alpha1Interface
+	// Deprecated: please explicitly pick a version if possible.
+	Nfs() nfsv1alpha1.NfsV1alpha1Interface
```

Comment:
I don't see a need to include this if it's already deprecated. We should just remove all of these deprecated functions from the codebase.

Reply:
@tonya11en this is generated code from k8s.io/code-generator.  If we delete this, it will just come back the next time `make codegen` is run.  Without an upstream code change, I don't see a smooth way to keep this non-versioned interface from being included.